Extensions/tf2_reblogs.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Tumblr Fortress 2 **//
-//* VERSION 1.1.1 **//
+//* VERSION 1.2.0 **//
 //* DESCRIPTION Replaces reblog icons with TF2 kill icons **//
 //* DEVELOPER circlejourney **//
```

Comment:
Shouldn't this be `new-xkit`?

Reply:
nah, until we have a better author/responsibility thing the original author keeps credit even if we're the ones actively maintaining it

Reply:
(except in the case where the original author is atesh or one of us, in which case you'd be right)